packages/.eslintrc.js

```diff
@@ -7,7 +7,21 @@ module.exports = {
     require.resolve('@contentful/eslint-config-extension/react'),
   ],
   rules: {
-    'react-hooks/exhaustive-deps': 'warn',
+    'react-hooks/exhaustive-deps': 'error',
+    'eslint-comments/require-description': [
```

Comment:
Ironically, I think we need a short description for the require-description rule haha. It's unclear what exactly are we requiring when looking at the `ignore` list